process_links.py

```diff
@@ -31,20 +31,6 @@
 )
 
 
-def show_inheritance(obj):
-    # handle inheritance printing to patch qgis._core with qgis.core
```

Comment:
This is not needed anymore?

Reply:
@3nids doesn't seem to be. I get the same result without it. Maybe it's related to moving this logic...

Reply:
Ok, cool then